extensions/quartz/runtime/src/main/java/io/quarkus/quartz/runtime/QuartzBuildTimeConfig.java

```diff
@@ -32,6 +33,9 @@
      * datasource configured. See <a href="https://quarkus.io/guides/datasource"> Configuring your datasource</a> for more
      * information.
      * <p>
+     * When configured with {@link StoreType#CUSTOM}, the value of quarkus.quartz.custom-store.class property have to implement
+     * <code>org.quartz.spi.JobStore</code>.
```

Comment:
...the value of quarkus.quartz.custom-store.class property _has to_ implement...

Reply:
corrected 'has'